### PR TITLE
Users features

### PR DESCRIPTION
**Pull request by Donkrzawayan**

### CODE REVIEW

File: Models/Models.csproj [hunk 1]
```diff
@@ -13,6 +13,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="7.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Identity.Core" Version="7.0.1" />
```
Comment (michcho415) [inline Models/Models.csproj:16]: Is it required?

File: Services/LandlordService.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        private readonly PasswordHasher<Landlord> hasher = new();
+
+        public LandlordService(IDbContextFactory<DatabaseContext> contextFactory)
+        {
+            this.contextFactory = contextFactory;
+        }
+
+        public async Task<IActionResult> CreateAsync(LandlordDTO landlord)
+        {
+            var existingLandlord = await GetAsync(landlord.Username);
```
Comment (michcho415) [inline Services/LandlordService.cs:21]: Email should also be validated

File: Services/LandlordService.cs [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+            if (existingLandlord != null)
```
Comment (michcho415) [inline Services/LandlordService.cs:22]: Email should also be validated.

File: Models/Models.csproj (hunk 1 above)
Comment (Donkrzawayan) [inline Models/Models.csproj:16]: yes, for PasswordHasher<>

Comment (michcho415) [inline Models/Models.csproj:16]: Is this package required?

**Review by michcho415 — APPROVED** [review]

**Pull request merged**